### ISSUE TITLE
Is there any certain way to colorize different ifc spaces in different colors?

### ISSUE THREAD

**Issue by turuncgil** [xeokit/xeokit-sdk#906]
Is there any possibility to assign different colors different elements in "IfcSpace" for example, i would like to give different colors to different rooms but cant think of a solution

**Comment by xeolabs** [comment]
This use case - setting per-type colors of each element within the structural hierarchy rooted by an IfcSpace - is supported by the xeokit Viewer. 

One possibility: once the model is loaded, you could use [getObjectIDsInSubtreeByType](https://xeokit.github.io/xeokit-sdk/docs/class/src/viewer/metadata/MetaObject.js~MetaObject.html#instance-method-getObjectIDsInSubtreeByType) to visit the child objects of a given IfcSpace and set each to a color based on its type.

That assumes that the IfcSpace is the root of whatever it contains.

Basically the  [MetaScene](https://xeokit.github.io/xeokit-sdk/docs/class/src/viewer/metadata/MetaScene.js~MetaScene.html) is designed to support these sort of ad-hoc visualizations.
